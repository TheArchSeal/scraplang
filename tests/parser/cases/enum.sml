enum e {}
enum e { x, y, z }
