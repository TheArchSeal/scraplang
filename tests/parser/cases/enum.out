stmt (13):1:1 enum e
stmt (13):2:1 enum e
    value   :2:10 x
    value   :2:13 y
    value   :2:16 z
